src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -63,30 +70,10 @@ private Value GetToolInfo(string toolName, string toolFileVersion, Version toolA
             var builder = ArrayBuilder<KeyValuePair<string, Value>>.GetInstance();
             builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.ToolName, toolName));
             builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.ToolAssemblyVersion, toolAssemblyVersion.ToString(fieldCount: 3)));
-            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.ToolFileVersion, GetToolFileVersionSubStr(toolFileVersion)));
+            builder.Add(CreateSimpleKeyValuePair(WellKnownStrings.ToolFileVersion, toolFileVersion));
             return Value.Create(builder.ToImmutableAndFree(), this);
         }
 
-        private string GetToolFileVersionSubStr(string toolFileVersion)
-        {
-            // Our log format specifies that fileVersion can have at most 3 fields.
```

Comment:
The spec says:

6.5.4. version property

A toolInfo object should contain a property named version whose value is a string containing the tool version. If present, the string should be in the format specified by Semantic Versioning 2.0.0. 

NOTE   Semantic versions have the property of being sortable in chronological order of release.

EXAMPLE   "1.0.0-beta"